src/lib/configs/project/fluence.ts

```diff
@@ -984,7 +984,7 @@ ${yamlDiffPatch(
 # customFluenceEnv:
 #   contractsEnv: local
 #   relays:
-#     - "${kras[0]?.multiaddr}"
+#     - "/ip4/127.0.0.1/tcp/10010/ws/p2p/12D3KooWGpqaLpVUoLB5cJPLDS6CjBNZePBNTYPGpNPNZMDs6osY"
```

Comment:
why?

Reply:
this is just an example comment anyway that most people will not ever see cause we turned off comments in configs by default. It's just simpler to have an example that doesn't unnecesserally rely on anything concrete (such as here where kras[0] can be undefined according to typescript even though we know it never is)